docs/TESTS.md

```diff
@@ -1,112 +1,65 @@
-## Xcode Tutorial
+## Testing Tutorial
 
-This guide explains how to create and configure a Swift-based Xcode project for Exercism.
+This guide explains how to run unit tests for Exercism Swift exercises, and optionally how to use Xcode for development.
 
-Apple does not support user-defined project templates in Xcode, and the default templates are more elaborate than those needed for Exercism. This guide bridges that gap.
+Exercism uses [Swift Package Manager](https://github.com/apple/swift-package-manager/tree/master/Documentation) to package files and tests for Swift.
```

Comment:
My aim isn't to preserve the work I did before. The new version is much shorter and easier to follow. I think its fine to wipe it out. I also agree that Mac-only is less than ideal. 

Perhaps explaining that this track works under both Mac and Linux near the top.I don't think that is explicit enough yet. 

Then break out the Xcode specific stuff to the bottom on the document.

I'd be happy to suggest a revision, if that's welcome.